pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -0,0 +1,199 @@
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React, { useState, useEffect } from 'react';
+import { Button, Flex, Modal, Popover } from "@patternfly/react-core";
+import { ExclamationTriangleIcon, ExternalLinkSquareAltIcon, HelpIcon } from '@patternfly/react-icons';
+
+import { ModalError } from 'cockpit-components-inline-notification.jsx';
+import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { TunedDialogBody } from '../../../pkg/tuned/change-profile.jsx';
+
+import './cryptoPolicies.scss';
+
+const _ = cockpit.gettext;
+
+// From /usr/share/crypto-policies/policies/$POLICY.pol
+const cryptopolicies = {
+    DEFAULT: _("A reasonable default for today's standards. It should provide 112-bit security with the exception of SHA1 signatures in DNSSec SHA1 is also enabled in HMAC where collision attacks do not matter."),
+    FUTURE: _("A level that will provide security on a conservative level that is believed to withstand any near-term future attacks. And also provide some (not complete) preparation for post quantum encryption support in form of 256 bit symmetric encryption requirement. It provides at least an 128-bit security. This level may prevent communication with many used systems that provide weaker security levels (e.g., systems that use SHA-1 as signature algorithm)."),
+    LEGACY: _("Provides settings for ensuring maximum compatibility with legacy systems. This policy is less secure and intended to be a easy way to switch system to be compatible with older systems. It should provide at least 80-bit security and excludes 3DES and RC4."),
+};
+const defaultpolicy = "DEFAULT";
+const stampfile = "/run/cockpit/crypto-policies-reboot-stamp";
+
+export const CryptoPolicyRow = () => {
+    const [showCryptoPolicy, setShowCryptoPolicy] = useState(false);
+    const [currentCryptoPolicy, setCurrentCryptoPolicy] = useState("");
+    const [isOpen, setIsOpen] = useState();
+
+    useEffect(() => {
+        // Avoid cockpit.spawn every re-render from parent
+        if (!showCryptoPolicy) {
+            cockpit.spawn(["update-crypto-policies", "--show"]).then(output => {
+                setCurrentCryptoPolicy(output.trim());
+                setShowCryptoPolicy(true);
+            });
+        }
+    }, [showCryptoPolicy]);
+
+    if (!showCryptoPolicy) {
+        return null;
+    }
+    const permission = cockpit.permission({ admin: true });
```

Comment:
This is old-fashioned. It doesn't react to superuser changes on the page. Please use pkg/lib/superuser.js instead.